### DISCUSSION TITLE
is there a way to add a dynamic variable in a portion of a rest endpoint ?

### DISCUSSION THREAD

**Discussion by MassoundiOmar** [Budibase/budibase#12252] [Help] [answered]
Hi 
I m looking for a way to call this rest api endpoint easily.
I would like to replace the queryId here with a dynamic variable.

My queryId is coming from a previous rest request. I configured a dynamic varible in order to use it accross my rest request.

rest endpoint : https://localhost/myexternalapi/queries/{queryId}/results

Is it possible please ?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @MassoundiOmar,

I've attached a screenshot of how I've done this.
<img width="955" alt="Screenshot 2023-11-02 at 13 34 02" src="https://github.com/Budibase/budibase/assets/126772285/b8fa5c48-d59f-414f-b76c-db59f20b094a">


I hope this helps.

**Reply by MassoundiOmar** [reply]
Thank you @ConorWebb96
